cmd/client/packet_injector.go

```diff
@@ -95,7 +99,9 @@ func addInjectPacketFlags(cmd *cobra.Command) {
 	cmd.Flags().StringVarP(&dstIP, "dstIP", "", "", "destination node IP")
 	cmd.Flags().StringVarP(&srcMAC, "srcMAC", "", "", "source node MAC")
 	cmd.Flags().StringVarP(&dstMAC, "dstMAC", "", "", "destination node MAC")
-	cmd.Flags().StringVarP(&packetType, "type", "", "icmp", "packet type: icmp4")
+	cmd.Flags().Int64VarP(&srcPort, "srcPort", "", 0, "source port for TCP packet")
+	cmd.Flags().Int64VarP(&dstPort, "dstPort", "", 0, "destination port for TCP packet")
+	cmd.Flags().StringVarP(&packetType, "type", "", "icmp4", "packet type: icmp4/6, tcp4/6")
```

Comment:
maybe better to give the exact options like:

packet type: icmp4, icmp6, tcp4, tcp6